pages/exhibition/_.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div
+    data-qa="exhibition page"
+  >
+    <HeroImage
+      v-if="hero"
+      :image-url="heroImage.url"
+      :image-content-type="heroImage.contentType"
+      :header="header"
+      :lead="lead"
+      :identifier="hero.identifier"
+      :citation="hero.citation"
+      :rights-statement="hero.license"
+      :name="hero.name"
+      :provider="hero.provider"
+      :creator="hero.creator"
+      :url="hero.url"
+    />
+    <b-container class="pb-3">
+      <b-row>
+        <b-col
+          cols="9"
+          class="pb-3"
+        >
+          <article>
+            <!-- eslint-disable vue/no-v-html -->
+            <div
+              data-qa="exhibition text"
+              v-html="mainContent"
+            />
+            <!-- TODO: remove when credits go to their own page? -->
+            <div
+              v-html="credits"
+            />
+            <!-- eslint-enable vue/no-v-html -->
+          </article>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col>
+          <!-- TODO: add links to chapters, remove h2 -->
+          <h2>Chapters</h2>
+          <ul v-if="page.hasPart">
+            <li
+              v-for="chapter in page.hasPart"
+              :key="chapter.fields.identifier"
+            >
+              {{ chapter.fields.name }}
+            </li>
+          </ul>
+        </b-col>
+      </b-row>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import marked from 'marked';
+  import createClient from '../../plugins/contentful';
+  import HeroImage from '../../components/generic/HeroImage';
+
+  export default {
+    components: {
+      HeroImage
+    },
+    computed: {
+      hero() {
+        return this.page.primaryImageOfPage ? this.page.primaryImageOfPage.fields : null;
+      },
+      header() {
+        return this.page.name;
+      },
+      lead() {
+        return this.page.headline;
+      },
+      heroImage() {
+        return this.hero ? this.hero.image.fields.file : null;
+      },
+      mainContent() {
+        return marked(this.page.text);
```

Comment:
As there are two computed methods here, which solely convert markdown to HTML (and will be plenty elsewhere), let's introduce a Vue filter for this so that in the template we can just do:
```html
            <div
              v-html="text | markup"
            />
```

Reply:
this doesn't work so nicely due to this: https://github.com/vuejs/vue/issues/4080